src/BladeTranslationsGenerator.php

```diff
@@ -42,6 +42,7 @@ public function generate(?string $locale = null, bool $wrapInHtml = true, bool $
 
     private function makeMaticeObject(?string $locale): string
     {
+        $locale = $locale ?? config('app.locale');
         $translations = json_encode($this->translations($locale));
         $appLocale = $locale ?? app()->getLocale();
```

Comment:
I understans that here you want to prevent the app from crashing when the local is missing. But what's the difference between line 45 and line 47? Can we just move line 47a bit up so that we can $appLocal as the default language for `$this->translations()` ?

Reply:
From what I see in code actually, the translations() function handles properly the case where the local is null. Can you please elaborate here how to reproduce the null local issue?